google-chrome-launcher

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
It always starts with a simple/innocent shell script... and then... :P Maybe it's fine. I would keep a low threshold for doing a Python ver if the complexity creeps up.